data/card_catalogue/unique-1.json

```diff
@@ -264,27 +184,17 @@
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Unarmed attacks deal double damage until you play your next card.",
```

Comment:
We don't have a notion of unarmed, so I've just put down attacks.

This card is insanely broken (or would be if the game was about fighting) but I like it

Reply:
![based](https://i.kym-cdn.com/entries/icons/original/000/031/015/cover5.jpg)